src/evm/prompts/implement-ecrecover-precompile.md

```
@@ -49,6 +49,58 @@ File: Search for `ecrecover` in evmone precompiles for gas costs and validation
+**Gas Constant** - `/go-ethereum/params/protocol_params.go` (line 140):
+```go
+EcrecoverGas        uint64 = 3000 // Elliptic curve sender recovery gas price
+```
+
+**Full Implementation** - `/go-ethereum/core/vm/contracts.go` (lines 264-301):
+```go
+// ecrecover implemented as a native contract.
+type ecrecover struct{}
+
+func (c *ecrecover) RequiredGas(input []byte) uint64 {
+	return params.EcrecoverGas
```

markdownlint-cli2 (0.17.2)

Hard tabs
Column: 1

(MD010, no-hard-tabs)